doc/README.rst

```diff
@@ -9,17 +9,20 @@ for both the stable and the latest (i.e., development) releases.
 Instructions
 ~~~~~~~~~~~~
 
-After installing NetworkX and its dependencies, install the Python
-packages needed to build the documentation by entering the root
-directory and executing::
+Install NetworkX along with its extra dependencies. For example, from
+the root of the repository:
 
-    pip install -r requirements/doc.txt
+```
+pip install .[extra]
+```
```

Comment:
Is there a particular reason why this isn't just indented like the other examples?
```suggestion
    pip install .[extra]
```

Or alternatively, if we want the examples to be copy-pastable, we should make _all of them_ code blocks.